homeassistant/components/growatt_server/sensor_types/inverter.py

```diff
@@ -33,135 +33,135 @@
         api_key="ipv3",
         native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
         device_class=SensorDeviceClass.CURRENT,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_wattage_input_3",
-        name="Input 3 Wattage",
+        translation_key="inverter_wattage_input_3",
         api_key="ppv3",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_internal_wattage",
-        name="Internal wattage",
+        translation_key="inverter_internal_wattage",
         api_key="ppv",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_reactive_voltage",
-        name="Reactive voltage",
+        translation_key="inverter_reactive_voltage",
         api_key="vacr",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_inverter_reactive_amperage",
-        name="Reactive amperage",
+        translation_key="inverter_reactive_amperage",
         api_key="iacr",
         native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
         device_class=SensorDeviceClass.CURRENT,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_frequency",
-        name="AC frequency",
+        translation_key="inverter_frequency",
         api_key="fac",
         native_unit_of_measurement=UnitOfFrequency.HERTZ,
         device_class=SensorDeviceClass.FREQUENCY,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_current_wattage",
-        name="Output power",
+        translation_key="inverter_current_wattage",
         api_key="pac",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_current_reactive_wattage",
-        name="Reactive wattage",
+        translation_key="inverter_current_reactive_wattage",
         api_key="pacr",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_ipm_temperature",
-        name="Intelligent Power Management temperature",
+        translation_key="inverter_ipm_temperature",
         api_key="ipmTemperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         precision=1,
     ),
     GrowattSensorEntityDescription(
         key="inverter_temperature",
-        name="Temperature",
+        translation_key="inverter_energy_today",
```

Comment:
This looks incorrect. It's a temperature sensor, not an energy sensor.

Reply:
Fixed with https://github.com/home-assistant/core/pull/95791